integration-tests/test-pipelines/parse-asset-extends/assets/country.sql

```diff
@@ -0,0 +1,14 @@
+/* @bruin
+name: country
+type: duckdb.sql
+materialization:
+   type: table
+
+extends: 
+  - Customer
+    
```

Comment:
what if we had some other columns, and some other extends maybe?
```suggestion
extends: 
  - Customer
 
columns:
  - name: mycol1
  - name: street_name
    extends: Address.Street
```